### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by jleslie85**
1. Updated and clarified text, shortened text in many places
2. Install showed a Helm 3 installation, but commands were still Helm 2. Converted Helm 2 commands to Helm 3 commands.
3. Added information about NuoDB Insights, description and how to install, and provided a link to the NuoDB Insights github

### CODE REVIEW

File: incubator/demo-ycsb/README.md [hunk 1]
```diff
@@ -146,7 +146,7 @@ replicationcontroller "demo-ycsb" scaled
 To uninstall/delete the deployment:
 
 ```bash
-helm del --purge ycsb
+helm del ycsb
```
Comment (sivanov-nuodb) [inline incubator/demo-ycsb/README.md:149]: Maybe better t use the full command version `helm delete ycsb`?

File: stable/README.md [hunk 2]
```diff
@@ -86,19 +83,30 @@ Each Helm Chart has a default values.yaml parameter file that contains configura
 
 ## Deployment Steps
 
-**Note:** You MUST first disable Linux Transparent Huge Pages(THP) on all cluster nodes that will host NuoDB pods. Run the `transparent-hugepage` chart first.
+1. Disable Linux Transparent Huge Pages(THP) on all cluster nodes that will host NuoDB pods.
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
```
Comment (sivanov-nuodb) [inline stable/README.md:88]: I guess these whitespaces are added by the editor.

**Review by sivanov-nuodb — APPROVED** [review]
Thanks for updating the docs and including crosslinks to Insights doc!

File: stable/README.md (hunk 2 above)
Comment (vegichan) [reply stable/README.md:88]: all of these should be removed

**Review by vegichan — APPROVED** [review]
looks good

File: incubator/demo-ycsb/README.md (hunk 1 above)
Comment (jleslie85) [reply incubator/demo-ycsb/README.md]: Yes, I think so too, thanks for your suggestion. I also changed in all other locations. "del" was used in all the chart clean-up sections.

File: stable/README.md (hunk 2 above)
Comment (jleslie85) [reply stable/README.md:88]: They were not added by the editor. I added them for indentation to format the text better.

**Pull request merged**